schedule/kernel/prepare_baremetal.yaml

```diff
@@ -0,0 +1,32 @@
+name:          prepare_baremetal
+description:    >
+    Conduct an installation on bare metal using IPXE with UEFI.
+vars:
+    DESKTOP: textmode
+    IPXE: 1
+    IPXE_CONSOLE: ttyS1,115200
+    IPXE_HTTPSERVER: http://baremetal-support.qa.suse.de:8080
+    IPXE_UEFI: 1
```

Comment:
We could even leave this one setting out and re-use this also for non-uefi installations, and define this variable on the machine level, as it depends on how the dhcp server is configured for the individual machine.

Reply:
This is correct, it should be at machine level and will make test suite more generic.